fast64_internal/sm64/settings/properties.py

```diff
@@ -115,9 +117,28 @@ def upgrade_version_1(self, scene: Scene):
         export_type = self.get("exportType", None)
         if export_type is not None:
             self.export_type = enum_export_type[export_type][0]
-
         self.version = 2
 
+    def upgrade_version_2(self, scene: Scene):
+        # props upgrade for combined export panel
+        combined_props = scene.fast64.sm64.combined_export
+        old_scene_props_to_new = {
+            "geoLevelName": "custom_export_name",
+            "geoExportPath": "custom_export_path",
+            "geoName": "object_name",
+            "geoGroupName": "group_name",
+        }
+        for old, new in old_scene_props_to_new.items():
+            setattr(combined_props, new, scene.get(old, getattr(combined_props, new)))
+        export_type = scene.get("geoExportHeaderType", None)
+        if export_type is not None:
+            combined_props.export_header_type = enumExportHeaderType[export_type][0]
+
+        level_name = scene.get("geoLevelOption", None)
+        if level_name is not None:
+            combined_props.level_name = enumLevelNames[level_name][0]
+        self.version = 3
+
```

Comment:
When you update to main you can use the upgrade_old_prop function i added, it has the multiple prop get the first one that used to be set functionality
for example
`upgrade_old_prop(self, "object_name", scene, {"geoLevelName", "colLevelName", "animLevelName"})`